cli/tests/environment-install.bats

```diff
@@ -145,8 +145,7 @@ teardown() {
   mkdir 2
   "$FLOX_BIN" init --dir 2
 
-  SHELL=bash run expect -d "$TESTS_DIR/install/prompt-which-environment.exp"
-  assert_success
+  SHELL=bash run -0 expect -d "$TESTS_DIR/install/prompt-which-environment.exp"
```

Comment:
no assert_success because expect ends with `exit 0` anyway?

Reply:
The "-0" handles that check.